@ornikar/eslint-config/plugins/unicorn.js

```diff
@@ -82,5 +82,23 @@ const formTree = shallow(tree.find(x => FormWithApiCall(x)).prop('children')());
 
     // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-exponentiation-operator.md
     'unicorn/prefer-exponentiation-operator': 'error',
+
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prevent-abbreviations.md
+    'unicorn/prevent-abbreviations': 'error',
+
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-for-loop.md
+    'unicorn/no-for-loop': 'error',
```

Comment:
c'est une bonne idée mais uniquement sur node, car babel le transforme en un bloc pas du tout performant

Reply:
Je comprend pas trop ce que tu veux dire en fait ? Si on autorise pas l'usage de ce type de boucle, Babel n'aura pas l'occasion de le transformer en un bloc peu performant justement ?

Reply:
cette regle demande de passer par un for-of plutot qu'un for

Reply:
https://babeljs.io/repl#?babili=false&browsers=&build=&builtIns=false&spec=false&loose=false&code_lz=FAMw9gTgBAFAxmAdgZwC5QKYBsMFsOLpghQCGEEpAngJRQDewAkAimDgHRZgDmM2eAqhoBuYAF8gA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=es2015%2Ces2016%2Ces2017%2Creact%2Cstage-0%2Cstage-1%2Cstage-2%2Cstage-3%2Ces2015-loose&prettier=false&targets=&version=7.4.2